CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-02-22
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-03 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on June 30th, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting this week on 22 February 2021 (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.2 for cluster 1.20.2 and above. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
+* Calico support in Windows is now supported as an add-on. To leverage this feature, AKS clusters will need to be on Kubernetes version 1.20 and above. Windows Calico support can only be enabled through cluster creation only. For more information, please see the [documentation](https://docs.microsoft.com/azure/aks/use-network-policies) on March 3rd 2021.  
+
+### Release Notes
+
+* Features
+  * Azure Application Gateway Ingress Controller add-on is now available in Government clouds.
+* Component Updates
+  * Calico updated to v3.17.2.
+  * AKS Windows image has been updated to [2019-datacenter-core-smalldisk-17763.1697.210210](vhd-notes/AKSWindows/2019/17763.1697.210210.txt).
+  * AKS Ubuntu 16.04 image updated to [AKSUbuntu-1604-2021.02.17](vhd-notes/aks-ubuntu/AKSUbuntu-1604/2021.02.17.txt).
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2021.02.17](vhd-notes/aks-ubuntu/AKSUbuntu-1804/2021.02.17.txt).
+
```

Comment:
As a user, it would be nice to have a full overview of ongoing deprecations
```suggestion

### Deprecations

* Azure Kubernetes Service legacy Azure AD integration is retiring on 29 February 2024
* AKS dashboard add-on will be removed from Kubernetes 1.19+
* Support for Pod Security Policies (preview) will no longer be available after June 30th, 2021 in Azure Kubernetes Service
```

I know PSP is mentioned in the announcement, but that's not the only deprecation

This is based on:
- https://github.com/azure-deprecation/dashboard/issues/127
- https://github.com/azure-deprecation/dashboard/issues/121
- https://github.com/azure-deprecation/dashboard/issues/62

Reply:
Hey @tomkerkhove,

Thanks for suggesting this! We will look into it!

Reply:
Guess it didn't make the cut 😁